data/guake-prefs.desktop.in

```diff
@@ -1,10 +1,7 @@
 [Desktop Entry]
 Encoding=UTF-8
-Name=Guake Preferences
-Name[pt_BR]=Preferências do Guake
-Comment=Configure your Guake sessions
-Comment[pt_BR]=Configurações do terminal Guake
-Comment[fr_FR]=Configurer votre session Guake
+_Name=Guake Preferences
+_Comment=Configure your Guake sessions
```

Comment:
Sorry indeed, I think I wasn't quite awake and my eyes were not quite open, so basically  -ENOCOFFEE :)